core/controllers/creator_dashboard_test.py

```diff
@@ -737,7 +738,7 @@ def test_can_update_display_preference(self):
         self.assertEqual(display_preference, 'list')
         self.logout()
 
-    def test_get_collections_list(self):
+    def test_can_create_collections(self):
 
         class MockCollection(object):
```

Comment:
Please delete MockCollection. We shouldn't need it.

Reply:
Done